configs/datasets/imagenav/gibson.yaml

```diff
@@ -0,0 +1,29 @@
+ENVIRONMENT:
```

Comment:
@mathfac How have we been using `configs/datasets`?  I am actually not sure what that folder is.

Reply:
We append task config with dataset one: `tasks/imagenav.yaml,datasets/imagenav/mp3d.yaml`. Worth to drop everything except `DATASET` section.